sky/packages/sky/lib/src/fn3/framework.dart

```diff
@@ -686,21 +687,27 @@ abstract class BuildableElement<T extends Widget> extends Element<T> {
 
   static BuildScheduler scheduleBuildFor;
 
-  static int _debugStateLockLevel = 0;
-  static bool get _debugStateLocked => _debugStateLockLevel > 0;
-
-  /// Calls the callback argument synchronously, but in a context where calls to
-  /// State.setState() will fail. Use this when it is possible that you will
-  /// trigger code in components but want to make sure that there is no
-  /// possibility that any components will be marked dirty, for example because
-  /// you are in the middle of layout and you are not going to be flushing the
-  /// build queue (since that could mutate the layout tree).
-  static void lockState(void callback()) {
-    _debugStateLockLevel += 1;
+  static int _debugBuildScopeDepth = 0;
+  static bool get _debugInBuildScope => _debugBuildScopeDepth > 0;
+
+  /// Establishes a scope in which component build functions can run
```

Comment:
Ah, you're right.  I thought I was following the style guide https://www.dartlang.org/articles/doc-comment-guidelines/ but it says to put a period after the first line.